spec/fabricators/keyword_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Fabricator(:keyword) do
+  on_init { init_with(FFaker::Identification.ssn, FFaker::FreedomIpsum.word) }
```

Comment:
```suggestion
  id { init_with(FFaker::Identification.ssn }
  name { FFaker::FreedomIpsum.word) }
```

If you use the attr_accessor as suggested above, then you can use a simpler and more explicit Fabricator ;-)

Reply:
I prefer to keep using `attr_reader` for this to satisfy the line, I can come back to look at this if this model requires change to `attr_accessor` 🙏